code/core/src/preview-errors.ts

```diff
@@ -295,6 +295,29 @@ export class TestingLibraryMustBeConfiguredError extends StorybookError {
   }
 }
 
+export class NoStoryMountedError extends StorybookError {
```

Comment:
Seems like this conflicts with https://github.com/storybookjs/storybook/pull/28434/files